modules/electrophysiology_browser/ajax/get_electrophysiology_session_data.php

```diff
@@ -0,0 +1,203 @@
+<?php
+/**
+ * Get EEG Session Data.
+ *
+ * This retrieves the entire eeg session data
+ * for a candidate to be visible on the eeg_session form.
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  EEG_Session
+ * @author   Muhammad Khan <muhammad.khan@mcin.ca>
+ * @license  Loris license
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+require_once 'BIDSFile.class.inc';
+
+/**
+ * User permission verification:
+ * Only users that have the eeg view all sites permission OR
+ * have the view site permission and belong to candidates session site
+ * are able to fetch the eeg session data.
+ */
+$user =& \User::singleton();
+$timePoint =& \TimePoint::singleton($_REQUEST['sessionID']);
+if (!$user->hasPermission('electrophysiology_browser_view_allsites')
+    && !((in_array($timePoint->getData('CenterID'), $user->getData('CenterIDs')))
+        && $user->hasPermission('electrophysiology_browser_view_site')
+    )
+)
+{
+    header('HTTP/1.1 403 Forbidden');
+    exit;
```

Comment:
I think we're trying to move away from the ajax exiting structure for modules. It involves a bit of refactoring so it probably shouldn't block this PR (especially since I can see you aren't the one who wrote it @cmadjar). But we should start not allowing this for new modules... at some point.

Reply:
True. Not sure how to refactor that for now though... Can be a separate PR maybe?